apps/web/.env.example

```
@@ -180,4 +180,8 @@ SINGULAR_WEBHOOK_TOKEN=
 
 # Hubspot
 HUBSPOT_CLIENT_ID=
-HUBSPOT_CLIENT_SECRET=
\ No newline at end of file
+HUBSPOT_CLIENT_SECRET=
+
+# Stablecoin payout
+STRIPE_FINANCIAL_ACCOUNT_ID=
+STRIPE_STABLECOIN_WEBHOOK_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)